Parental Leave (#50)
flowforge/handbook: peopleops/index.md

manager updated on your health, and let them know what FlowForge can do for you
to aid in your recovery.

## Parental Leave

If you live in a country where a statutory Parental Leave benefit is available,
you will be required to claim statutory Parental Leave pay (if you are eligible).

Any employee can take up to 6-weeks if they wish without requiring manager approval. We recommend a minimum of 4 weeks, but recognise it's a personal choice to make. Further leave can be requested, but should be discussed with your manager.

> If you live in a country that offers statuatory leave longer than this, then

Author: @ZJvandeWeg I tried to cover statutory maternity leave with this statement, for example, in the UK:
- 90% salary for the first 6 weeks (we improve on this with full pay)
- £151.97 or 90% of your average weekly earnings (whichever is lower) for the next 33 weeks (we could offer full pay for more weeks up front), but would otherwise offer statutory.

manager updated on your health, and let them know what FlowForge can do for you
to aid in your recovery.

## Parental Leave

If you live in a country where a statutory Parental Leave benefit is available,
you will be required to claim statutory Parental Leave pay (if you are eligible).

Whilst statutory parental leave will vary from country-to-country, at FlowForge

> we have a mandatory 4-week, full-pay, Parental Leave policy. Any employee is

Reviewer: My only hesitation, what if people want a distraction from a baby? Maybe mandatory is to firm, and we should go with:

```suggestion
we advise you to take 4-weeks, full-pay, Parental Leave policy. Any employee is
```

Reviewer 2: Given we already have an unlimited leave policy how is this 4 weeks any different?

Reviewer 3: Maybe flip it around...

> Any employee can take up to 6-weeks is they wish without requiring manager approval. We recommend a minimum of 4 weeks, but recognise its a personal choice to make. Further leave can be requested, but should be discussed with your manager

Reviewer 3: @sammachin As per handbook, anything over 2 weeks needs approval. This is an exception to that - parental leave means you have up to 6 weeks without needing approval.

Reviewer: @knolleary I think your proposal is a good improvement and we should update the text to that.

> Whilst statutory parental leave will vary from country-to-country, at FlowForge
> we have a mandatory 4-week, full-pay, Parental Leave policy. Any employee is
> able to extend this to 6-weeks if they wish, without requiring manager approval.
> Further leave can be requested, but should be discussed with your manager.

Reviewer: ```suggestion
Any employee can take up to 6-weeks is they wish without requiring manager approval. We recommend a minimum of 4 weeks, but recognise its a personal choice to make. Further leave can be requested, but should be discussed with your manager
```

Reviewer: @joepavitt ^ is a one-click update of the PR per: https://github.com/flowforge/handbook/pull/50/files#r828038014
